content/docs/atom-guide/look-dev/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,669 @@
+
+      // Overall wind
+      float4 currentBending = setUpWindBending(currentTime, worldPosition);
+
+      // Detail bending
+      float3 currentOutPosition = detailBending(IN.m_position, IN.m_normal, IN.m_optional_color, currentTime, worldPosition, currentBending.w);
+
+      currentOutPosition = mainBending(currentOutPosition, currentBending);
+
+      worldPosition = mul(objectToWorld, float4(currentOutPosition, 1.0));
+   }
+   ```
+5. Repeat steps 3-4 with the depth pass in `o3de\Gems\Atom\Feature\Common\Assets\Materials\Types\VegetationBending_DepthPass.azsli` and the depth pass with PS in `o3de\Gems\Atom\Feature\Common\Assets\Materials\Types\VegetationBending_DepthPass_WithPS.azsli`.
+6. Open the **Editor** and you should see your tree swaying with the leaves still bending. If you don't, try opening the **Material Editor** and increasing the wind parameters.
+
+Amazing, our tree now sways and reacts to wind! Try to place multiple trees and observe how the trees sway differently when close together versus farther away. Also, add some lighting to make the trees pop!
+
+{{< video src="/images/atom-guide/vegetation-bending-tutorial/tree3.mp4" autoplay="true" loop="true" width="100%" muted="true" info="Video of trees swaying." >}}
+
+### Add motion vectors
+Because our tree is moving, we can add a *motion vector pass* so we can choose to add motion blur or other effects in the future.
+
+If you take a look at the `MeshMotionVectorVegetationBending.azsli` file, you can see that the vertex shader looks similar to the other vertex shaders we have seen, but there's a new output `OUT.m_worldPosPrev`. This is the position of the vector in the last time frame. The pixel shader uses both the previous vector position and the current one to calculate the motion vector. 
+
+However, we don't have an vertex shader input that gives us the previous position. Therefore, in the vertex shader, we want to calculate the bending for not only the current time as we have been, but also for the previous time frame.
+
+1. Download the three files in the `MeshMotionVectorVegetationBending` folder and move it to `o3de\Gems\Atom\Feature\Common\Assets\Materials\Types`.
+2. The `MeshMotionVectorVegetationBending` files already contain all the previous steps that we have done, like adding the shader option and calling the bending functions. 
+2. Open `VegetationBending.materialtype`. Add the bottom of the `shaders` list, add the motion vector pass:
+   
+   ```
+   {
+      "file": "./MeshMotionVectorVegetationBending.shader",
+      "tag": "MeshMotionVector"
+   }
+   ```
+3. Remember the `.lua` files and how we could enable/disable shaders? Let's ensure our motion vector shader is enabled. Open `VegetationBending_ShaderEnable.lua`.
+4. At the bottom of the `Process` fuunction, outside of any conditionals, enable the motion vector shader:
+   
+   ```
+   local motionPass = context:GetShaderByTag("MeshMotionVector")
+   motionPass:SetEnabled(true)
+   ```
```

Comment:
Is there anything that disables the motion vector pass? I didn't see anything. If not, then you can remove this step, as all shaders are enabled by default.

Reply:
Yeah, I originally wanted to just place this to show how to simply edit the lua file, but I'll probably delete this section since I'm planning on taking your previous suggestion to demonstrate the lua functor with the converting the angle to x, y values.